### ISSUE TITLE
Create User for login with different permissions

### ISSUE THREAD

**Issue by MoBlockbuster** [louislam/uptime-kuma#1322]
### ⚠️ Please verify that this feature request has NOT been suggested before.

- [X] I checked and didn't find similar feature request

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

- Possibility to create non-admin user with only view permission
- Possibility to create manager user with permission to create checks but not to modify settings

### ✔️ Solution

- currently no solutions

**Comment by Prajwalprakash3722** [comment]
Yes this feature can be added :)

**Comment by MoBlockbuster** [comment]
awesome :) thanks

**Comment by athornfam2** [comment]
Would this request include that "new" user getting their own dashboard or private status page view?

**Comment by MoBlockbuster** [comment]
exact. The "new" user/group has his own dashboards with his monitoringchecks and statistics.
so you can create checks for user A and some other checks for user B. 
With help of restrictions the checks for user A is not visible for user B

**Comment by athornfam2** [comment]
Sure would be nice to get this feature added. I'm not a dev but seems like it shouldn't be too hard to add-in...

**Comment by acki** [comment]
+1

**Comment by Magnusemail** [comment]
Adding a read only user would be very useful, is this possible today - if so how ?

**Comment by ririko5834** [comment]
I like this, admins should be able to make new users and manage them (log in as them, change password for them, delete them). The user should be able to view just their monitors and status pages, admins should be able to login as other users to view and manage their monitors

**Comment by bobthedev90368** [comment]
Please!

**Comment by keriati** [comment]
Multi user / user rights management would be a nice addition.

**Comment by rmarops** [comment]
I'd like to see read only user for end users and operators to have visibility to any monitors created by admin.

**Comment by dkornahrens** [comment]
I, also, need more user accounts.

**Comment by Teffin-Varghese** [comment]
Yes, It would be great if you guys can integrate multi-user capability with different privileges on this awesome tool.

**Comment by d4rklynk** [comment]
Agreed to too, would love to see that feature.

**Comment by bitsky6** [comment]
Is there any update regarding this?

**Comment by darkkhaoz** [comment]
Multi user would be cool so they could manage and be alerted by their own sites

**Comment by RafaBlockDev** [comment]
Any 2025 update about this?
